toolkit/tools/pkg/licensecheck/licensenames.go

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+// A tool for validating %license entries in rpms
+
+package licensecheck
+
+import (
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+)
+
+// licenseNamesFuzzy is a list of license names that should be matched in a case-insensitive sub-string search
+var licenseNamesFuzzy = []*regexp.Regexp{
+	regexp.MustCompile(`(?i).*copying.*`),
```

Comment:
Would it make sense to move these names into resource file under `toolkit/resources/manifests/package` as well?

Reply:
Not a bad idea, will do.